next.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
   reactStrictMode: true,
```

Comment:
Why do we need this?

Reply:
I didn't change that...
Git got confused
That is part of the default next config file

Reply:
ahh i understand. I being a dummy.